src/component/admin/groups/EditGroup/EditGroup.tsx

```diff
@@ -65,6 +70,22 @@ export const EditGroup = () => {
         navigate(GO_BACK);
     };
 
+    const onSetName = (name: string) => {
+        clearErrors();
+        if (
+            groups?.filter(group => group.name === name && group.id !== groupId)
```

Comment:
Could be extracted for clarity

Reply:
Addressed in https://github.com/Unleash/unleash-frontend/pull/1210/commits/84c6a37dcbe94b9eb34951200484f07fa3345cd5